bin/e2e-test-client/src/tests/script.rs

```diff
@@ -143,34 +139,12 @@ pub async fn run_contract_large_state(ctx: &TestContext) -> Result<(), Failed> {
 }
 
 pub async fn arbitrary_transaction(ctx: &TestContext) -> Result<(), Failed> {
-    const RAW_PATH: &str = "src/tests/test_data/arbitrary_tx.raw";
```

Comment:
I removed this because I don't see what it has to do with test and it was blocking my update.

If we want to add some roundtrip conversion test somewhere between raw/json, that should be separate from this logic.

Reply:
e2e binary is not a test. It is a binary that runs some tests on a real network. One of the tests is to run an arbitrary transaction where the transaction is hex-encoded from the canonical representation.

The source of the transaction should be `raw` file, not `json` file, because it is how the user sends transaction to the blockchain(in a `raw` representation). The idea of the tool is that you can take any transactions submitted to GraphQL and dry run it on a real network.

I agree that we need to document the use case of the `e2e` binary better and describe what it is responsible for. But we don't need to remove the logic that we need.